src/Core/Mesh/TopologicalTriMesh/OpenMesh.h

```diff
@@ -1,55 +1,45 @@
 #ifndef TOPOLOGICALMESH_OPENMESH_H
 #define TOPOLOGICALMESH_OPENMESH_H
 
-#define RELEASE_EIGEN_PLUGIN_PROTECTION_AGAINST_DIRECT_INCLUSION
-#define EIGEN_MATRIX_PLUGIN "Core/Mesh/TopologicalTriMesh/EigenOpenMeshPlugin.h"
-
 #include <Core/RaCore.hpp>
 
 #include <Eigen/Core>
 #include <Eigen/Geometry>
 
 // Need to define openMesh namespace function before including openmesh, for
 // MSVC
```

Comment:
This comment could be made more explicit. Why do we need this ?

Reply:
@nmellado could you be more specific about it?

Reply:
ok, what about:
```c++
// We use Eigen::Matrix as base type to represent matrices in OpenMesh.
// This is not standard, and several namespace functions are not available in OpenMesh
// for Eigen::Matrix. 
// We need these functions to be defined before including OpenMesh, otherwise some 
// compilers (e.g. MSVC) will not be able to compile the code.
```

Reply:
What about the inclusion of `OpenMesh.h`?

Reply:
I don't understand your question.

Reply:
File `src/Core/Mesh/TopologicalTriMesh/OpenMesh.h` should be included before any OpenMesh include right?

Reply:
I'm not sure this is required, as the failure is caused by the compilation of the TopologicalMesh.

Reply:
OK for this comment.